tests/unit/plugin.js

```diff
@@ -268,6 +269,24 @@ define([
                     done();
                 });
             });
+
+            it('does not close item when header clicked', function(done) {
+                $element.bellows();
+
+                var $openDisabledItem = $(openDisabledItem);
+
+                $element.bellows('add', $openDisabledItem);
+
+                $openDisabledItem
+                    .find('.bellows__header')
+                    .trigger('click');
+
+                setTimeout(function() {
```

Comment:
@scalvert I was copying the test structure of MQ's test above it. Most likely reason why MQ had it in his test was to re-queue execution (which I doubt is necessary) or if the tests fails, to allow for Bellows' animation. 

TL:DR; It was like that when I got here. :P